src/main/java/core/basesyntax/strategy/impl/BalanceFruitAmountCalculateStrategy.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.FruitAmountCalculateStrategy;
+
+public class BalanceFruitAmountCalculateStrategy implements FruitAmountCalculateStrategy {
```

Comment:
let's simplify all the names in this package to -> BalanceHanlder, PurchaseHandler etc